### PR TITLE
WIP for BXMSDOC-4119-master: Created a content for how to run a single test scenario.

### PR DESCRIPTION
**Pull request by kaldesai**
[Original Epic](https://issues.jboss.org/browse/BXMSDOC-4007)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-4119)
[Test Scenario RHPAM enterprise doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4119-RHPAM-TS/#test-scenarios-intro-con)
[Test Scenario RHDM enterprise doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4119-RHDM-TS/#test-scenarios-intro-con)

Please review the following:
Chapter 1. Test scenarios --> 3rd Paragraph
Section 3.10. Contextual menu options --> Table 3.1 and 3.2

Thank you.

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-intro-con.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ Test scenarios in {PRODUCT} enable you to validate the functionality of business
...
 
-With the test scenarios designer, you execute all scenarios from the `.scesim` file at one time, whereas with the legacy test scenarios designer, you can execute test scenarios one at a time or as a group. The group execution contains all the scenarios from one package. Test scenarios are independent, so one scenario cannot affect or modify the other. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios.
+The defined test scenarios in both the designers can be executed in multiple ways, for example you can run the all the available test scenarios at a project level or inside a particular test scenario asset. For more information, see xref:test-designer-run-test-proc[]. Test scenarios are independent, so one scenario cannot affect or modify the other. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-intro-con.adoc]: Do we use "execute" or "run" when referring to running test scenarios?

Number of ways is more common in the docs:
s/multiple ways/number of ways

Typo:
s/run the all the/run all the

s/modify the other./modify another./modify other test scenarios.

s/particular/specific/

I rewrote some of the paragraph, please feel free to adopt the parts you agree with:

You can execute the defined test scenarios in a number of ways, for example, you can run available test scenarios at the project level or inside a specific test scenario asset. Test scenarios are independent and cannot affect or modify other test scenarios. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios. For more information about running test scenarios, see xref:test-designer-run-test-proc[].

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
+. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon to execute all the test scenarios from the table at once. To run a single test scenario, right click on the row that you want to execute and select *Run scenario*.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc]: "Execute" or "run"?

Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc]: "Run" is preferred.
These steps are already updated in the BXMSDOC-4115 JIRA.

[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc]: "Run" is preferred.
These steps are already updated in the BXMSDOC-4115 JIRA.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-intro-con.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-intro-con.adoc]: Thank you for the detailed review. 
I will incorporate the typo and suggested changes.

**Pull request closed without merging**